owncloudApp/src/main/java/com/owncloud/android/presentation/files/createfolder/CreateFolderDialogFragment.kt

```diff
@@ -82,12 +87,48 @@ class CreateFolderDialogFragment : DialogFragment() {
         alertDialog.setOnShowListener {
             val okButton = alertDialog.getButton(AlertDialog.BUTTON_POSITIVE)
             okButton.isEnabled = isButtonEnabled
+
+            okButton.setOnClickListener {
+                var fileName: String = inputText.text.toString()
+                if (fileName.isEmpty()) {
+                    error = getString(R.string.uploader_upload_text_dialog_filename_error_empty)
+                } else {
+                    null
+                }
```

Comment:
I think this case is not reachable, since the OK button is always disabled by default when the filename is empty (empty string or string made up by spaces), so we'll never be able to click on it, we can remove this code.
But anyway, given that code, the `else` branch is doing nothing right now. Maybe you wanted to write:
```suggestion
                error = if (fileName.isEmpty()) {
                    getString(R.string.uploader_upload_text_dialog_filename_error_empty)
                } else {
                    null
                }
```
In this case, the `else` block does makes sense 👍

Reply:
Done